pythonforandroid/recipes/python3/patches/fix-ctypes-util-find-library.patch

```diff
@@ -10,9 +10,16 @@ diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
 +if True:
 +    def find_library(name):
 +        # Check the user app libs and system libraries directory:
-+        app_root = os.path.normpath(os.path.abspath('../../'))
-+        lib_search_dirs = [os.path.join(app_root, 'lib'), "/system/lib"]
-+        for lib_dir in lib_search_dirs:
++        from jnius import autoclass
```

Comment:
sorry to ask, but that should probably be an optional dep with a warning and fallback to previous behavior.
```python
try:
    from jnius import autoclass
    # same things as you did below
except ImportError:
    app_root = os.path.normpath(os.path.abspath('../../'))
    native_lib_dir = os.path.join(app_root, 'lib')
```
Well you get the idea. I don't like that my suggestion adds complexity, but I'm always worrying about adding new hard deps.

Reply:
@AndreMiras if ARM64 wasn't *the* main target for the future I would agree. but ARMv7 is going to grow less and less important anyway, right? (in which case making this dep optional won't help many people in the medium-term future) Do we want to carry around this additional complexity for the 2-3 years it still somewhat matters, hoping we don't forget to remove it later?

I personally would rather leave it out, but as always I leave the decision up to you guys